doc/examples/ex03/ex03.sh

```diff
@@ -19,41 +19,38 @@
 #
 
 gmt begin ex03
```

Comment:
We decided to consistently use tabs for indentations in the example scripts. Please change the indentations to tabs.

Reply:
OK, done. This version predated the tab decision..